TODO.md

```diff
@@ -0,0 +1,9 @@
+# Shit to do
+- how can I write tests for logging in?
```

Comment:
I think in most cases I don't actually see tests testing OAuth, but what we could probably do to test these is to capture the response back from a legitimate OAuth request and store it in the tests (or in a text fixture) and just use that as the "response" in the test, rather than making an actual API call, and then make sure our handling goes through and creates/logins the actual user. I think Devise exposes some helper methods for checking signed-in users, but it's been a little while since I've worked with this stuff.